deployments/ocis-nats/helmfile.yaml

```diff
@@ -7,13 +7,46 @@ releases:
     chart: nats/nats
     namespace: ocis-nats
     values:
-      - cluster:
-          enabled: true
-          name: "ocis-cluster"
-      - nats:
+      - config:
+          cluster:
+              enabled: true
+              replicas: 3
+              name: "ocis-cluster"
           jetstream:
             enabled: true
 
+      - config:
```

Comment:
It really works that way with having `config` twice? I would expect one entries with the subentries merged.

Reply:
Yes, it worked. This is probably a thing of helmfile because it's internally transforming this to `helm apply` parameters. 

I'll fix it because it looks confusing.